example/src/Strada/Form.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from 'react-native';
+import React from 'react';
+import { BridgeComponent } from 'react-native-turbo';
+import { NavigationContext } from '@react-navigation/native';
+
+export default class Form extends BridgeComponent {
+  static name = 'form';
+  static contextType = NavigationContext;
+
+  constructor(props) {
+    super(props);
+  }
+
+  setHeaderButton(disabled: boolean) {
+    this.context.setOptions({
+      headerRight: () => (
+        <Button
+          onPress={() => this.replyTo('connect')}
+          title="Send"
```

Comment:
The Strada Web component provides a `submitTitle` as the message data, so let's use that in the form component as well:

```
// iOS
private func handleConnectEvent(message: Message) {
    guard let data: MessageData = message.data() else { return }
    configureBarButton(with: data.submitTitle)
}
```